config/settings.py

```diff
@@ -165,7 +168,6 @@
 
 STATIC_URL = 'static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
-STATICFILES_DIRS = (os.path.join(BASE_DIR, 'static'),)
```

Comment:
Accidentally added this in my last pr to try to fix local admin styles unsuccessfully